drivers/include/net/netdev2.h

```diff
@@ -10,15 +10,153 @@
  */
 
 /**
- * @defgroup    drivers_netdev_netdev2 Generic Low-Level Network Driver Interface v2
+ * @defgroup    drivers_netdev_netdev2 Network Device Driver Interface
```

Comment:
I changed this line to

```
 * @defgroup    drivers_netdev_api Network Device Driver API
```

in #6610. But I guess it is okay to change this here and to change it in #6610. Just keep in mind to remove this change if #6610 gets merged first.